docker-compose-build.yml

```diff
@@ -66,6 +76,12 @@ services:
       default:
         aliases:
          - wdqs.svc
+    environment:
+      - WIKIBASE_HOST=wikibase.svc
+      - WDQS_HOST=wdqs.svc
+      - WDQS_PORT=9999
+    expose:
+      - 9999
```

Comment:
Why is 9999 now also being exposed?

Reply:
Previously it was always exposed (hardcoded to 9999) in the dockerfile.

I moved it out of there because the port that needs to be exposed is defined by the WDQS_PORT

Reply:
As we said in IRC I'm not sure if we should take this out.

It seems rather unreasonable to force users to set this variable on WDQS updater if they can't actually change it in our images.

If we do take it out we should then also code it as the default in WDQS updater and remove the requirement of setting it there.

Reply:
Yup, I think I was confusing "ports" and "expose" in the docker-compose file when I last looked at this.